consensus/src/round_manager.rs

```diff
@@ -420,6 +450,20 @@ impl RoundManager {
         Ok(())
     }
 
+    fn sync_only(&self) -> bool {
+        if self.decoupled_execution {
+            let back_pressure = self.back_pressure.load(Ordering::SeqCst);
+            let root_round = self.block_store.root().round();
+            counters::OP_COUNTERS
```

Comment:
this should be decoupled into two metrics, this is my quick hack to know the back pressure value

Reply:
resolved

Reply:
we need to store the sync_only flag in both branches, something like
let sync_only = if .. {
} else {
};

counters.set(sync_only);
sync_only